### PR TITLE
Pgdb dsl2

### PR DESCRIPTION
**Pull request by DongdongdongW**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
 - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/pgdb/tree/master/.github/CONTRIBUTING.md)
 - [ ] If necessary, also make a PR on the nf-core/pgdb _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -35,26 +39,19 @@ jobs:
...
         env:
           CAPSULE_LOG: none
         run: |
           wget -qO- get.nextflow.io | bash
           sudo mv nextflow /usr/local/bin/
       - name: Run pipeline with test data
-        # TODO nf-core: You can customise CI pipeline run tests as required
         # For example: adding multiple test runs with different parameters
         # Remember that you can parallelise this by using strategy.matrix
         run: nextflow run ${GITHUB_WORKSPACE} -profile test,docker
```
Comment (ypriverol) [inline .github/workflows/ci.yml]: Currentlty we are testing just a few use cases. We must included more examples including testing COSMIC downloaded etc.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -35,26 +39,19 @@ jobs:
             environment.yml
       - name: Build new docker image
         if: env.MATCHED_FILES
-        run: docker build --no-cache . -t nfcore/pgdb:dev
```
Comment (ypriverol) [inline .github/workflows/ci.yml:38]: Do we need this here, in DSL2 I don't think we need to add the docker containers here. The nextflow will do it. Each module will download their own container or tool.

File: modules/local/CanonicalAndNoncanonicalProteomes/add_ncrna.nf [hunk 3]
```diff
@@ -0,0 +1,25 @@
+/**
+ * Creates the ncRNA protein database
+ */
+process ADD_NCRNA {
```
Comment (ypriverol) [inline modules/local/CanonicalAndNoncanonicalProteomes/add_ncrna.nf]: Can you name all the modules with lower case. **the folders I mean**. The process itself can be uppercase.

**Review by ypriverol — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (DongdongdongW) [reply .github/workflows/ci.yml:38]: I removed the code that built the container.

**Pull request merged**